packages/file_selector/file_selector_web/test/test_driver/utils_integration.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.9
+
+import 'dart:html';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:file_selector_web/src/utils.dart';
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+
+void main() {
+  group('FileSelectorWeb utils', () {
+    IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+    group('acceptedTypesToString', () {
```

Comment:
Can this group be in a unit test, instead of an integration one? Or does it complain of importing dart:html?